app.rb

```diff
@@ -0,0 +1,30 @@
+require_relative 'database'

```

Comment:
I'm getting an error message from this line which means I can't open the app through Ruby or Rackup, and also makes your rspec tests fail. The error message is:
"undefined method `/' for #<String:0x00007ff4d1b1dcf8>"